lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -358,9 +361,10 @@ def _prepare_environment(  # noqa: C901
                     for azure_cap in location_info.capabilities:
                         if azure_cap.vm_size == node_runbook.vm_size:
                             predefined_cost += azure_cap.estimated_cost
-
-                            min_cap: schema.NodeSpace = req.generate_min_capability(
-                                azure_cap.capability
+                            min_cap: schema.NodeSpace = (
+                                self._azure_generate_min_capability(
```

Comment:
Because this method is in AzurePlatform, it doesn't need the keyword `Azure`. Maybe it can be called like `_node_generate_min_capability`?

Reply:
Sure! I will update this.